### PR TITLE
Adds a delete confirmation dialog

### LINKED ISSUE #1754: Window to confirm user wants to delete slide, or option to undo delete slide action

### PR DESCRIPTION
**Pull request by javierarce**
This PR adds a confirmation dialog before removing a slide.

Fixes #1754

screen shot 2015-04-20 at 17 29 43

### CODE REVIEW

**Comment by javierarce** [comment]
@viddo: can you review this, please?

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3128/))

File: lib/assets/javascripts/cartodb/table/slides/panel.js [hunk 1]
```diff
@@ -295,7 +295,37 @@ cdb.admin.SlideView = cdb.core.View.extend({
...
+      ok_title: "Yes, do it",
+      cancel_button_classes: "underline margin15",
+      cancel_title: "Cancel",
+      modal_type: "confirmation",
+      width: 500
+    });
+
+    this.delete_confirmation = delete_confirmation;
+    this.delete_confirmation.on('clean', function _c() {
+      self.delete_confirmation.unbind('clean', _c);
```
Comment (viddo) [inline lib/assets/javascripts/cartodb/table/slides/panel.js]: Why is this necessary?

Comment (javierarce) [inline lib/assets/javascripts/cartodb/table/slides/panel.js]: Ups, it's not needed. Removed.

**Comment by viddo** [comment]
Looks straight forward, so :+1: 
I suppose there is some basic test there already?

Btw, thanks for notifying, I'll add it to the list of modals to keep in mind when implementing the news ones. This is for odyssey, right?

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3131/))

**Comment by javierarce** [comment]
Yes, this is just for odyssey. 

There are tests for the slide model and panel, but I didn't add it for this particular feature.
